.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.5', '2.6', '2.7']
+        ruby: ['2.6', '2.7']
```

Comment:
For now mutant internally always use the ruby parser that is based on the current ruby version.

Attempting to mutation cover branches that are only required for `> 2.5` would fail. Instead mutant needs a mechanism of a "target ruby". Than even 2.5 could pass the mutation test on > 2.5 syntax.

For now this is the easier way to get the bugfix in.